### PR TITLE
TagKey should be BIGINT

### PR DESCRIPTION
**Pull request by pengchen0692**
## Description
@TagKey is a variable to save tagKey, so that it should be as same type as TagKey. this is a typo.

### CODE REVIEW

**Review by snehabandla — APPROVED** [review]

File: src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.diff.sql [hunk 1]
```diff
@@ -249,7 +249,7 @@ AS
         
         -- Check if tag exsit
         DECLARE @tagStatus TINYINT
-        DECLARE @tagKey TINYINT
+        DECLARE @tagKey BIGINT
```
Comment (smithago) [inline src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.diff.sql:252]: Can you add a little bit more reasoning on why the change in the description. Will we ever support 8bytes, 2power63 different custom tags.

Comment (pengchen0692) [reply src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.diff.sql:252]: @tagKey is a variable to save tagKey, so that it should be as same type as TagKey. this is a typo.
